mail_filter_adressee_by_contact/__manifest__.py

```diff
@@ -9,6 +9,7 @@
     "category": "Social Network",
     "website": "https://github.com/OCA/social",
     "author": "Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["Kev-Roche"],
     "license": "AGPL-3",
     "application": False,
     "installable": True,
```

Comment:
add sale dependency if you want to leave `record.partner_shipping_id` call on `_get_domain_for_account_move`

Reply:
Can't target the depends line, so I put the comment in this line